connect/cli/plugins/project/extension/helpers.py

```diff
@@ -60,6 +75,22 @@ def bootstrap_extension_project(config, output_dir, overwrite):  # noqa: CCR001
     if not overwrite and os.path.exists(project_dir):
         raise ClickException(f'The destination directory {project_dir} already exists.')
 
+    if save_answers_dir:
+        if os.path.exists(save_answers_dir):
+            console.secho(
+                f'File {save_answers_dir} already exists. Answers will not be saved.',
+                fg='red',
+            )
+        elif os.path.splitext(save_answers_dir)[-1] != '.json':
```

Comment:
I would not check the provided extension

Reply:
But in this case user can save it to another format and in future will not be able to use it to load answers. And will not understand why....

Reply:
But the extension does not guarantee the format